.github/workflows/ci.yml

```
@@ -0,0 +1,34 @@
+# Notes:
+#  - All software installed below is pinned so they don't get upgraded and break w/o us knowing why.
+#  - If you bump one version up to latest, please bump them all so we stay current!
+
+name: Build medic-couch2pg and run unit test cases
+
+on: [push, pull_request]
+
+jobs:
+  build-and-test:
+    name: Build medic-couch2pg
+    runs-on: ubuntu-18.04
+
+    steps:
+    - name: Checkout branch
+      uses: actions/checkout@v2
+
+    - name: Use nodeJS 10.x
```

This is more about setting up, right?
```suggestion
    - name: Setup NodeJS
```

---

Done